package.json

```diff
@@ -74,9 +67,8 @@
     "ethr-did-resolver": "^8.0.0",
     "hardhat": "^2.14.0",
     "hardhat-gas-reporter": "^1.0.9",
-    "jsonschema": "^1.4.1",
-    "jsonwebtoken": "^9.0.0",
-    "key-did-resolver": "1.4.0",
+    "jose": "^5.2.4",
```

Comment:
The [jsonwebtoken](https://github.com/auth0/node-jsonwebtoken) library is targeting node enviroments. I replaced it with https://github.com/panva/jose